src/components/inputs/text/ValidationInput/ValidationInput.tsx

```diff
@@ -42,7 +43,13 @@ function ValidationInput({
   value: string | number | undefined;
   setValue: (text: string) => void;
   validityCallback?: (validity: boolean) => void;
-  validationPredicates: { [x: string]: (value: string) => Promise<any> };
+  validationPredicates: {
+    [x: string]: {
+      fn: (value: string) => Promise<any>;
+      required?: boolean;
+      message?: string;
```

Comment:
message isn't needed - you can use the key for the message like we were doing previously

Reply:
updated in https://github.com/ar-io/pdns-react/pull/209/commits/982e3592adf3cab1fa9871675d369e4198b0ae80